bench/spawn.go

```diff
@@ -239,3 +275,56 @@ func (cmd *SpawnCommand) spawnRemote(ctx context.Context) ([]BenchmarkResult, er
 
 	return results, nil
 }
+
+// getGithubContent using github api return url of a file to download and commit hash
+func (cmd *SpawnCommand) getGithubContent(ctx context.Context) (*GitContent, error) {
```

Comment:
I'd prefer passing filename and repo and having this be a stand alone function rather than a method of SpawnCommand